index.html

```diff
@@ -5,50 +5,57 @@
+                <ul>
+                    <li><a href="#portfolio">Portfolio</a></li>
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                </ul>
+            </nav>
         </header>
-        <section class="welcome display-flex flex-column space-arround align-middle">
+        <section class="welcome">
             <h1 class="greetings txt-center neutrals">Hey there. I'm Utsargo</h1>
             <h1 class="greetings txt-center success">I'm a micronaut at Microverse</h1>
             <p class="txt-center neutrals">
                 I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
 don’t hestiate to contact me.
             </p>
-            <ul class="social-links col-md-12 display-flex flex-row space-around align-middle">
+            <ul class="social-links">
                 <li><a href="https://twitter.com/royutsargo"><i class="fa-brands fa-twitter"></i></a></li>
                 <li><a href="https://www.linkedin.com/in/utsargo-roy/"><i class="fa-brands fa-linkedin-in"></i></a></li>
                 <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
                 <li><a href="https://github.com/utsargo"><i class="fa-brands fa-github"></i></a></li>
                 <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
             </ul>
-            <a href="#" class="down-arrow display-flex">
+            <a href="#" class="down-arrow">
                 <i class="fa-solid fa-chevron-down"></i>
             </a>
         </section>
-        <section class="recent-works display-flex flex-column align-middle">
+        <section class="recent-works" id="portfolio">
             <h2 class="section-heading txt-center">
                 My Recent Works
             </h2>
             <div class="separator">
 
             </div>
-            <div class="work-list">
-                <div class="card portfolio-item display-flex flex-column padding-0 txt-center align-middle">
+            <div class="cards-showcase">
+                <div class="card portfolio-item">
                     <img src="./assets/images/portfolio-4.png" alt="Multi-Post Stories Gain+Glory" class="thumbnail">
-                    <div class="portfolio-text display-flex flex-column txt-center align-middle">
+                    <div class="portfolio-text">
                         <h3 class="card-title">
                             Multi-Post Stories Gain+Glory
                         </h3>
-                        <ul class="categories display-flex align-middle justify-center">
+                        <ul class="categories">
                             <li><a href="" class="category">Ruby on rails</a></li>
                             <li><a href="" class="category">css</a></li>
                             <li><a href="" class="category">Javascript</a></li>
```

Comment:
Please notice that these elements shouldn't be `a` tags 😉. As displayed in Figma:
![image](https://user-images.githubusercontent.com/42799579/199645547-de2b3d61-75ae-4a3e-aaf9-abebca84298b.png)

You can still keep the hover effect, it looks very cool 😀.

> Note: you did an impressive job with the design! Kudos to you, well done 😄! Keep up the good work, you will be an amazing and very talented developer, I'm sure 😉.